src/configuration/types.rs

```diff
@@ -271,6 +271,8 @@ pub struct Configuration {
   pub jsx_quote_style: JsxQuoteStyle,
   #[serde(rename = "jsx.multiLineParens")]
   pub jsx_multi_line_parens: JsxMultiLineParens,
+  #[serde(rename = "jsx.forceContentNewLine")]
+  pub jsx_force_content_new_line: bool,
```

Comment:
This also needs an update in `configuration/builder.rs` then in that file add an entry in the `check_all_values_set` test.